### ISSUE TITLE
NoClassDefFoundError: … R$styleable circlepageIndicator.java

### ISSUE THREAD

**Issue by anantshah93** [JakeWharton/ViewPagerIndicator#379]
how to solve error

Caused by: android.view.InflateException: Binary XML file line #43: Error inflating class com.viewpagerindicator.CirclePageIndicator

Caused by: java.lang.NoClassDefFoundError: com.app.skedule.R$styleable
10-03 13:24:46.541 31183-31183/? E/AndroidRuntime:     at com.viewpagerindicator.CirclePageIndicator.<init>(CirclePageIndicator.java:93)

after changing 
 dependencies {
        classpath 'com.android.tools.build:gradle:1.0.0'
    }

to 

 dependencies {
        classpath 'com.android.tools.build:gradle:1.3.0'
    }

**Comment by ccrousp** [comment]
Hi,
did you happen to find a solution to that issue?
Charles

**Comment by anantshah93** [comment]
Yes, But now I don't know How i did it. It's long time before occurred. But thanks You at-least  asked....

**Comment by javieramp** [comment]
Hi!
Check to remove this line: this works for me --> app:fillColor="@color/xxx" at XML
